### PR TITLE
added email verification functionality

### PR DESCRIPTION
**Pull request by owais635**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```
Comment (owais635) [inline gradlew:1]: @TheOGApplePie Why are these Gradle files being comited?

File: lib/main.dart [hunk 2]
```diff
@@ -127,7 +127,7 @@ class MainScreen extends StatelessWidget {
         return SplashScreen();
 
       case Status.Unauthenticated:
-        return SignInOptionsScreen();
+        return SignInOptionsScreen();
```
Comment (owais635) [inline lib/main.dart:130]: @TheOGApplePie  please remove the extra space.

Comment (owais635) [reply lib/main.dart:130]: just reverse the changes made to this file.

File: local.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (owais635) [inline local.properties:1]: It says this `This file must *NOT* be checked into Version Control Systems`. So why is this being committed? lol

**Comment by aashir-khan** [comment]
Can some images/videos be added to the PR to demonstrate the changes?

File: lib/main.dart (hunk 2 above)
Comment (aashir-khan) [reply lib/main.dart:130]: I'm confused, who made the PR?

**Pull request closed without merging**